shopinvader_search_engine_image/schemas/product_category.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.shopinvader_product.schemas import ProductCategory as BaseCategory
+
+from .images_mixin import ImageMixin
+
+
+class ProductCategory(BaseCategory, ImageMixin, extends=True):
+    @classmethod
+    def from_product_category(cls, odoo_rec):
+        obj = super().from_product_category(odoo_rec, with_parent=True, with_child=True)
```

Comment:
@lmignon Here is the bug on category bindings.

```suggestion
        obj = super().from_product_category(odoo_rec)
```

Reply:
Thank you @qgroulard